util/topsql/reporter/reporter.go

```diff
@@ -552,7 +553,7 @@ func (tsr *RemoteTopSQLReporter) doReport(data reportData) {
 		return
 	}
 
-	agentAddr := variable.TopSQLVariable.AgentAddress.Load()
+	agentAddr := config.GetGlobalConfig().TopSQL.ReceiverAddress
```

Comment:
Does this mean that we will not allow dynamically changing it at all?

Reply:
Yes.